### ISSUE TITLE
Can we ran DeepMimic directly in DeepMimicCore's main.cpp?

### ISSUE THREAD

**Issue by Zju-George** [xbpeng/DeepMimic#70]
@xbpeng @wangshub @0xflotus @376TEMP 
Also, if I want to debug and change the code, I must change C++ code?

I wonder, if I want to write the pose data out into a json file when glutMainloop is running,  should I change the C++ code and ran it directly in main.cpp in DeepMimicCore?

And after debugging in C++ code, rewrap it using SWIG?

I'm a noob in both C++ and python, please give me some advice.

**Comment by ZhengyiLuo** [comment]
Yes, you need to change the behavior of the bullet environment you need to change C++ code and recompile. If you are using windows in the sln there should be a python target you can build the python wrapper. 

You can not run the policies in main.cpp I think, since the policies can only be run in python using tensorflow. 

Please stop spamming other threads when you have build questions. Specifically: [This one](https://github.com/xbpeng/DeepMimic/issues/66)

**Comment by xbpeng** [comment]
That's right, you can edit the C++ code in DeepMimicCore and debug in C++. Then when you are ready you can run SWIG to build the python wrappers and train using the python framework.

You cannot run  the policies in C++ because all of the RL framework is in python.

**Comment by Zju-George** [comment]
yes we can do that, but it seems that we can't get the trained model run properly, but for the sake of running mocap data, it's fine. I will dig into more, thx anyway.
![image](https://user-images.githubusercontent.com/33959089/62026714-5eb95000-b20e-11e9-9070-a6f3ff7b08f5.png)

**Comment by Zju-George** [comment]
@xbpeng Thanks for your reply!
Could you please share some more ground types cuz there is only "dull" plane..
![image](https://user-images.githubusercontent.com/33959089/62261096-bc37e180-b446-11e9-8bb0-defdddf6ec9a.png)
